ajax/libs/ng-toast/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "ng-toast",
+  "filename": "ngToast.min.js",
+  "version": "2.0.0",
+  "author": "Tamer Aydin (http://tamerayd.in)",
+  "autoupdate": {
+    "fileMap": [
+      {
+        "basePath": "dist",
+        "files": [
+          "**/*"
+        ]
+      }
+    ],
+    "source": "git",
+    "target": "git://github.com/tameraydin/ngToast.git"
+  },
+  "description": "Angular provider for toast notifications",
+  "homepage": "http://tameraydin.github.io/ngToast",
```

Comment:
I am not sure. These were the URLs that author has supplied in the `package.json` file. I feel like we should preserve whatever the author wants the URLs to be.